src/Repository/PostRepository.php

```diff
@@ -36,31 +35,24 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Post::class);
     }
 
-    public function findLatest(int $page = 1, Tag $tag = null): Pagerfanta
+    public function findLatest(int $page = 1, Tag $tag = null): Paginator
```

Comment:
this is not a good idea to pass directly to Twig like that. The Countable implementation of the Paginator gives you the total number of results, not the number of items in the iterator itself. This will confuse twig (`loop.last` will become wunky for instance).